test/services/feed/post.logic.test.ts

```diff
@@ -44,7 +44,7 @@ describe("Get one post", () => {
 describe("Save post", () => {
   it("should save post to db and return whether it was successful", async () => {
     // Arrange
-    const postCreateDto = getPostCreateDtoMock();
+    const postCreateDto = inputMocks.getPostCreateDtoMock();
     const savedPost = getPostMock();
```

Comment:
estos metodos de mocks estan todos creados desde cero por vos no?

Reply:
no uso ninguna librería ahí, pero me inspire en este post: [https://itnext.io/mocking-in-typescript-a97267f7cea9](url).